sqs_workers/queue.py

```diff
@@ -29,6 +30,7 @@
 
 DEFAULT_MESSAGE_GROUP_ID = "default"
 SEND_BATCH_SIZE = 10
+MAX_SQS_MESSAGE_SIZE = 262144  # in bytes
```

Comment:
Maybe length? Same for the variable used below.

Reply:
Also drop the SQS since its redundant.